components/gitpod-cli/cmd/prebuild-logs.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/gitpod-io/gitpod/gitpod-cli/pkg/theialib"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// initCmd represents the init command
+var prebuildLogs = &cobra.Command{
+	Use:   "prebuild-logs",
```

Comment:
How about `gp prebuilds list` + `gp prebuilds log` to list human readable names of tasks and then fetch content.

Reply:
Currently `prebuild-logs-X` contains lot of things from binary characters to even process completion % (see [picture here](https://github.com/gitpod-io/gitpod/pull/13505#issuecomment-1264415276)). That's why I was saying to avoid printing in terminal.

Reply:
I think they are special characters for shell and should be interpreted properly. We don't write anything else to these files besides what was produced by pty service for shell.